client/views/network.tpl

```diff
@@ -1,5 +1,5 @@
 {{#each networks}}
-<section id="network-{{id}}" class="network" data-id="{{id}}" data-nick="{{nick}}" data-options="{{toJSON serverOptions}}">
+<section id="network-{{id}}" class="network name-{{slugify name}} net-{{slugify serverOptions.NETWORK}}" data-id="{{id}}" data-nick="{{nick}}" data-options="{{toJSON serverOptions}}">
```

Comment:
I'm thinking that, say in my theme I want to display a little Rizon-specific logo for Rizon, I can make it show on `net-rizon` and no matter what the user names the network in their client, it'll show up appropriately. I think that'd be the most common use case for themes making use of the network name.